files/galaxy/tpv/tools.yml

```diff
@@ -1305,17 +1310,17 @@ tools:
         - singularity
 
   toolshed.g2.bx.psu.edu/repos/iuc/funannotate_annotate/funannotate_annotate/.*:
+    inherits: basic_docker_tool
     cores: 8
-    mem: 40
+    mem: 42
+    params:
+      docker_run_extra_arguments: "--mount type=tmpfs,tmpfs-size=2147483648,destination=/tmp"
     env:
```

Comment:
Not trying to be pedantic, but this env related to singularity is also not required as this is now using docker.